### DISCUSSION TITLE
CDT Call November 2024

### DISCUSSION THREAD

**Discussion by jonahgraham** [eclipse-cdt/cdt#937] [Announcements]
Our next CDT Call will be November ~13th~ 20th at 11am Ottawa time. Please make a comment to add suggestions for discussion topics, or come along to the call and we can discuss anything in "Any other business" section. Everyone is welcome.

## Call Information

<details><summary>Call Information and Agenda</summary>
<p>

- **Location:** [:computer: Zoom Meeting](https://eclipse.zoom.us/j/545364607?pwd=L1d2eDQ4RWJmdkk3YXkvcW90RWJiQT09) ([full details below](#Connection-Details))
- **Date:** November 20, 2024 11:00 AM Eastern Time (US and Canada) ([Calendar/ICS](https://eclipse.zoom.us/meeting/uJAodO-sqDstovAF50NQR9k2f8kNjAOvEg/ics?icsToken=98tyKuysqzgsHtKVt1z9c6ktOcH-b9_yln5Yi4UIzDHUMXlYMAWuPcxIE51pNd-B)) ([world clock](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2024&month=11&day=13&hour=16&min=0&sec=0&p1=188&p2=136&p3=195))
- **Agenda**
1. Welcome
2. Actions from last meeting
3. CDT 12.0 planning
4. Any other business?
- **Contact:** <cdt-dev@eclipse.org> or [start a discussion topic](https://github.com/eclipse-cdt/cdt/discussions)
- **Reference:** [Previous and future meetings](https://github.com/eclipse-cdt/cdt/discussions?discussions_q=label%3Acdtcalls) (older call notes are located [here](https://hackmd.io/@jonahgraham/EclipseCDTCalls))

## Connection Details

Join Zoom Meeting
https://eclipse.zoom.us/j/545364607
Meeting ID: 545 364 607
Passcode: 196590
Dial by your location find your local number: https://eclipse.zoom.us/u/acMY3kyoJW

</p>
</details> 

## Minutes

Present: @ruspl-afed, @ghentschke, @jld01, @ewaterlander 

- Discussion on CDT-specific code comprehension features for Core Build projects. Agreement that any changes required to build commands to support code comprehension might be best placed in a separate build command fragment file that could be included by the principal build command file (eg CMakeLists.txt) if required.
- Discussion on release of CDT 12.0.0 in February 2025. It is desirable to prioritize any changes required to allow easy evaluation of CDT-LSP. Possible checkbox in project wizard to enable CDT-LSP.

**Comment by jonahgraham** [comment]
I will try to be in attendance to start the meeting this week - but the meeting falls in the middle of [TheiaCon](https://www.eclipse.org/events/2024/theiacon/) and I am presenting "Zero-Install Embedded C/C++ Development: Running GDB in the Browser with WebAssembly" at TheiaCon at the same time as the CDT call.

**Reply by ruspl-afed** [reply]
@jonahgraham others may be also interested to attend https://www.eclipse.org/events/2024/theiacon/ and see your presentation live. Shouldn't we shift the meeting to resolve this conflict?

**Reply by jonahgraham** [reply]
That would simplify my life and there are some interesting talks at TheiaCon (Patrick Chuong is talking about CCS too). Lets delay to the 20th.
